src/components/Dashboard.js

```diff
@@ -16,8 +17,9 @@ const Dashboard = (props) => {
                 <img className="review-image" src={item.review_img_url} alt={item.title}/>
                 <p>{item.title}</p>
                 <p class="designer">{item.designer}</p>
-                <p>Review by {item.owner}</p>
-                <p class="review-text">{item.review_body}</p>
+                <Link to={`/review/${item.review_id}`}>
```

Comment:
I'd suggest extracting out the info from line 16 into it's own component for readability!